src/batchify.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Creates set of operations which need to be applied to the input in order to transform
+ * it to the output. Can be used with strings or arrays.
+ *
+ *		const input = Array.from( 'abc' );
+ *		const output = Array.from( 'xaby' );
+ *		const batch = batchify( diff( input, output ), output );
+ *
+ *		batch.forEach( operation => {
```

Comment:
I do not like the collision name with batch and operation. Especially because you will use it in the same place where you will use engine batch, so you will have 2 thinks called `batch`.